index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+      <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Poetry</title>
+  </head>
+   <body style="background-color:lightblue;">
+  <body>
+    
+    <p>
+      <h1> 
+  <strong>ON PAPER</strong>
+    </h1> 
+    </p>
+    
+<p>
+ <h2>
+   By Jacqueline Woodson
+  </h2> 
+    </p>
+
+ <p>
+   The first time I write my full name
+    </p>
+<p>
+  Jacqueline Amanda Woodson
+    </p>
+ <p>
+   without anybody's help
+    </p>
+ <p>
+   on a clean white page in my composition notebook,
+    </p>
+ <p>
+   I know  
+     </p>
+<p>
+   if I wanted to
+</p>
+<p>
+  I could write anything.
+    </p>
+<p>
+  Letters becoming words, words gathering meaning,
+    </p>
+<p>
+  becoming thoughts outside my head
+    </p>
+<p> <!--  🧠 🗣 -->
+    </p>  
+<p>
+  becoming sentences
+    </p>
+<p>
+  <p class="written-by">
+  <strong>Written by</strong>
+    </p>
+
+<p>
+  <p class="jac">
+  Jacqueline Amanda Woodson
+</p>  
+
+
+<img style="float: right;" alt="src=https://rockysmith.files.wordpress.com/2020/07/woodson-j.jpg>
```

Comment:
A handful of notes here:

1. See my earlier note about inline styles.
1. You're floating this image, but floating is really appropriate for when you want text to move around the image.  Instead, I would consider wrapping the photo and the website URL in a `footer` element and setting `text-align:right` on that, which would send all the text and the image inside of it to the right. See [this example](https://glitch.com/edit/#!/remix-henny-p1), which I remixed from your Project 1.
1. In general, you want to avoid linking to images or other resources on web sites belonging to other people. This is called **hotlinking** and we'll talk more about it in class or upcoming readings.
1. Your `img` element needs to be adjusted slightly:

```suggestion
<img alt="" src="https://rockysmith.files.wordpress.com/2020/07/woodson-j.jpg">
```

Here, I've removed the inline styles, wrapped the `src` URL in double quotes and made sure the `alt` and `src` attributes don't run into each other. We'll talk more soon about how to use the `alt` attribute.